operator/pkg/securedcluster/values/translation/translation.go

```diff
@@ -171,6 +173,14 @@ func (t Translator) checkInitBundleSecret(ctx context.Context, sc platform.Secur
 	return nil
 }
 
+func (t Translator) getGlobalMonitoring(m *platform.GlobalMonitoring) *translation.ValuesBuilder {
+	openshiftMonitoring := translation.NewValuesBuilder()
+	openshiftMonitoring.SetBoolValue("enabled", m.IsOpenShiftMonitoringEnabled())
+	globalMonitoring := translation.NewValuesBuilder()
+	globalMonitoring.AddChild("openshift", &openshiftMonitoring)
+	return &globalMonitoring
+}
+
```

Comment:
Can you extract this in a common translation package? Seems like this code is duplicated for `Central` and `SecuredCluster`

Reply:
Fixed